pbs/pbsrequest.go

```diff
@@ -83,10 +84,10 @@ type PBSRequest struct {
 	Device        *openrtb.Device `json:"device"`
 
 	// internal
-	Bidders []*PBSBidder      `json:"-"`
-	UserIDs map[string]string `json:"-"`
-	Url     string            `json:"-"`
-	Domain  string            `json:"-"`
+	Bidders []*PBSBidder `json:"-"`
+	UserIDs []string     `json:"-"`
```

Comment:
Assuming we don't get a large number of bidders, I figured the array would be more efficient performance wise. Since it is just an array of string pointers, the unused bidder IDs only take up a few bytes each. Not sure what the overhead of implementing a map is, but it might end up taking less memory than the map solution at the current size of the possible FamilyNames. If we suspect the number of families will grow significantly, then perhaps a map may be the better solution.